.github/workflows/source/hasNonASCII

```diff
@@ -28,6 +28,7 @@ for i in $(find . \
                 -not -path "*wp_parse*"        \
                 -not -path "./tmp_build_dir/*" \
                 -not -path "./Docs/*"          \
+                -not -path "./setup.py"        \
```

Comment:
I don't see any non-ascii in setup.py - why is it excluded?

Reply:
I wrote Maxence Thévenet and Rémi Lehe in the author/maintainer strings :)

Surprisingly, the `é` is not part of ASCII.

Reply:
Ah. Their names are written with `e` in the source code, so maybe be consistent with that rather than have this one exception.

Reply:
Can do, I just felt fancy because UTF8 works so well in Python :) @RemiLehe @MaxThevenet any preferences? This is the meta-data showing up on PyPI.